### PR TITLE
week-8-a

### PR DESCRIPTION
**Pull request by LeoTan1212**

### CODE REVIEW

**Comment by LeoTan1212** [comment]
PNG file for the Markdown
control
image

independent
Markvo
8-spr2024/
PDF
assets/169318855/bd0beeb2-4782-4ae7-bce3-b41105aebfa1)

**Comment by LeoTan1212** [comment]
control

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+- Example of Probability Mass Function (PMF): A discussion and example of PMFs, which apply to discrete random variables and give the probability that a discrete random variable is exactly equal to some value.
+
+- Example of Probability Density Function (PDF): A discussion and example of PDFs, which apply to continuous random variables and describe the likelihood of a random variable to take on a particular value.
+
+- Rules Corresponding to Random Variables: An overview of some fundamental rules and properties related to random variables.
+
+### Random Variable
+
+### Symbol
+- X:random varible(RV). a quantity whose value is unknown/uncertain
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: 'random varible' should be 'random variable'.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+### Probability Space ($\Omega$,$F$,$\mathcal{P}$)
+- $\Omega$: sample space (all possible outcomes of RV)
+- $\mathcal{F}$: Event space (all possible sets of outcomes in $\Omega$)
+- $\mathcal{P}$: Probability measure (assigns a probability to each other)
+
+## Moving forward consider X v P(X) where P(x) is a probabilty distribution
+- Expectation $\mathbb{E}$
+  
+  ${\mathbb{E} \text{xp} [g(x)]} = \int_x g(x) P(x) \, dx \text{ or } \sum_x g(x) P(x)$
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: The notation for expectation E should be clearly explained, including the integrals and sums for continuous and discrete cases, respectively.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  - mean $\mathcal{u_x}$ = ${\mathbb{E} \text{xp}(x) [g(x)]}$
+
+
+
+### Discrete Random Variable Plot
+
+The following plot represents the probability mass function (PMF) of a discrete random variable: 
+![Discrete Random Variable Plot](PMF.png)
+- $S=[1.2.3]$
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: ’$S=[1.2.3]$‘ should be ’$S = {1, 2, 3}$‘

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+  
+### The probability density function of a multivariate normal (or Gaussian) distribution
+$N(u, \Sigma) = \frac{1}{(2\pi)^{k/2} \det(\Sigma)^{1/2}} \exp\left(-\frac{1}{2}(x - u)^T \Sigma^{-1} (x - u)\right)$
+
+- $u$: the mean vector
+- $\Sigma$:the covariance matrix
+- $k$:the dimensionality of the distribution
+- $x$:the vector of the variables
+- $det(\Sigma)$:the determinant of the the covariance matrix 
+- $\Sigma^{-1}$is the inverse of the covariance matrix
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan:62]: Good point, I have clearly recognized the meaning of the probability density function here.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+$P(X,Y,Z)=P(X|Y,Z)P(Y,Z)=P(X|Y,Z)P(Y|Z)P(Z)$
+
+## Bayes‘ Rule
+$P(X|Y)P(Y)=P(Y|X)P(X)$
+
+$P(X|Y)=\frac{P(Y|X)P(X)}{P(Y)} \int_x P(Y|X)P(X) dx$
+
+$P(X|Y) \space \propto \space P(Y|X)P(X)$ ($\propto$: is proportional to)
+
+## Posterior, if Y is a measurement then what is X given Y?
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: 'if Y is a measurement then what is X given Y?' It seems a little unclear here.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+## Prior, your belief guess of X before knowing Y?
+### Likelihood Function
+ The joint probability mass (or probability density) of observed data is viewed as a function of the parameters of a statistical model.[2]
+- Given X, what you expect Y to be
+- What happens if we have multiple measures?
+- $P(x|y,y_2) \space \propto \space P(y_2|x,y_1)P(x|y_1)$
+
+Assume measurements are independent
+- - $\propto \space P(y_2|x)P(x|y_1)$
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan:108]: '$\propto \space P(y_2|x)P(x|y_1)$' should be explained properly in context.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+A general probabilistic approach for recursively estimating an unknown probability density function (PDF) over time using incoming measurements and a mathematical process model.[2]
+
+Recursive Bayes Estimation Example:
+-- Python Coding
+-- [Recursive Bayes Estimation](https://github.com/ankit-1517/Recursive-Bayesian-Estimation/blob/main/bayes_filter.ipynb)[3]
+
+
+$x$ is (static)
+
+$P(x|y_{1：k})=P(y_k|x)P(x|y_{1:k-1})=\int_x P(y_k|x)P(x|y_{1:k-1})dx$
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: This formula shows how the posterior at step k becomes the prior for step k + 1.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+
+Assume the system is Markov
+- "Next state depends only on the current state"
+- $P(x_0,x_1,x_2,..,x_k)=P(x_0) \pi ^k _{i=1}$P(x_i|x_{i-1})$ Note: no control input
+
+(Markvo Chain Rule) 
+
+![](Markvo.png)
+
+include control input
```
Comment (MarcoZhou6) [inline bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: "Include control input" should clearly explain the role of control inputs in a Markov Decision Process (MDP).

**Review by MarcoZhou6 — CHANGES_REQUESTED** [review]
On the whole, the content is very interesting and careful, and the professor's class notes are also very detailed. There are only some small problems, which will be better if corrected in time.

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 1 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: Thank you! It is helpful

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 2 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: Fixed. Thank you!

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 3 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: Fixed. Thank you!

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 5 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: Added some new content. Thank you!

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 6 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan:108]: Thank you!

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 7 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: Fixed. Thank you!

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 8 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan]: Added some content. Thank you!

File: bite-size-notes/week-8-a-Probability Review-Zhijie Tan (hunk 4 above)
Comment (LeoTan1212) [reply bite-size-notes/week-8-a-Probability Review-Zhijie Tan:62]: Thank you!

**Pull request merged**